### DISCUSSION TITLE
Vercel does not deploy my PR-preview

### DISCUSSION THREAD

**Discussion by augustinanders** [neuefische/questions#560] [Web] [answered]
I created my first Pull-Request before deploying my Project on Vercel. Now I deployed the project, but the vercel-bot does not respond under my Pull-Request. I don't know how I can deploy from my new branch manually. Can someone help me out?

**Comment by toshydev** [comment, marked as answer]
Did you try pushing a new commit onto the new undeployed branch?

**Reply by augustinanders** [reply]
how can I send a new commit, if i dont change the code? 
see here:
```
~/w/2023_03_01_capstone-neuefische_memory userstory_1_memory_grid ❯ git add .                                23:19:02
~/w/2023_03_01_capstone-neuefische_memory userstory_1_memory_grid ❯ git commit -m "new commit"               23:19:11
On branch userstory_1_memory_grid
Your branch is up to date with 'origin/userstory_1_memory_grid'.

nothing to commit, working tree clean
~/w/2023_03_01_capstone-neuefische_memory userstory_1_memory_grid ❯ git push                                 23:19:15
Everything up-to-date
```

**Reply by augustinanders** [reply]
ok thank you solved the problem by simply committing a test change. ...reversed it afterwards.
